systems/llnl-matrix/system.py

```diff
@@ -163,15 +163,15 @@ def compute_packages_section(self):
                     "externals": [{"spec": "curl@7.61.1", "prefix": "/usr"}],
                     "buildable": False,
                 },
+                "mpi": {"buildable": False},
```

Comment:
@scheibelp Can I define this here? If somehow MPI is not enabled, will defining the MPI virtual break things?

Reply:
Yes I meant outside of the MPI definitions. Putting it in the MPI definitions would avoid any confusion, but would require defining it for each MPI, so if that isn't necessary I think this is better.